### ISSUE TITLE
TypeError: Cannot read property 'charCodeAt' of undefined

### ISSUE THREAD

**Issue by steals** [forcedotcom/salesforcedx-vscode#1684]
### Summary

Can't provide the exact steps to reproduce the issue, but it happens from time to time when I save Aura Controller / Aura Helper.

```
TypeError: Cannot read property 'charCodeAt' of undefined
    at isInteger (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:547:18)
    at exports.Obj.hasProp (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:581:11)
    at findExprType (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:914:28)
    at findTypeAtExpr (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:927:16)
    at Object.findTypeAt [as run] (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:951:22)
    at run (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:275:59)
    at /Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:278:16
    at Object.exports.withContext (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:883:18)
    at /Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:274:13
    at analyzeAll (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:404:15)
[Error - 11:53:20 AM] TypeError: Cannot read property 'charCodeAt' of undefined
    at isInteger (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:547:18)
    at exports.Obj.hasProp (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:581:11)
    at findExprType (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:914:28)
    at findTypeAtExpr (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:927:16)
    at Object.findTypeAt [as run] (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:951:22)
    at run (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:275:59)
    at /Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:278:16
    at Object.exports.withContext (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/infer.js:883:18)
    at /Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:274:13
    at analyzeAll (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-46.15.1/node_modules/aura-language-server/lib/tern/lib/tern.js:404:15)
```

Might be regression of the https://github.com/forcedotcom/salesforcedx-vscode/issues/1356

**VS Code Version**: 1.38.1

**VS Code Salesforce Extension Version**: 46.15.1

**SFDX CLI Version**: 7.25.3

**OS and version**: MacOS 10.15

**Comment by steals** [comment]
Still reproduced on **VS Code Salesforce Extension Version** 47.1.0

**Comment by steals** [comment]
**VS Code Salesforce Extension Version 47.9.0**
```
[Error - 7:37:00 PM] (node:95530) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '0' of undefined
    at Object.indexStaticResources (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-47.9.0/node_modules/lwc-language-server/lib/metadata-utils/static-resources-util.js:55:85)
    at async Promise.all (index 2)
    at async Promise.all (index 0)
    at async Timeout._onTimeout (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-47.9.0/node_modules/aura-language-server/lib/server.js:39:9)
[Error - 7:37:00 PM] (node:95530) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '0' of undefined
    at Object.indexStaticResources (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-47.9.0/node_modules/lwc-language-server/lib/metadata-utils/static-resources-util.js:55:85)
    at async Promise.all (index 2)
    at async Promise.all (index 0)
    at async Timeout._onTimeout (/Users/steals/.vscode/extensions/salesforce.salesforcedx-vscode-lightning-47.9.0/node_modules/aura-language-server/lib/server.js:39:9)
```

**Comment by lcampos** [comment]
Closing this since the fix is available on v50.7.0, thanks for providing feedback.
